src/_pytest/fixtures.py

```diff
@@ -534,6 +534,8 @@ def getfixturevalue(self, argname: str) -> Any:
         :raises pytest.FixtureLookupError:
             If the given fixture could not be found.
         """
+        # Note: This is called during setup for evaluating fixtures defined via
```

Comment:
I found the comment somewhat confusing, would remove it.

Reply:
I definitely think there should be some comment here, or have the docstring modified*, because the docstring heavily implies that this function is only used when fixture functions are evaluated dynamically. That confused me a lot when trying to understand the code.
\* I'm guessing the docstring is for the public API documentation, so whether it's used internally or not probably shouldn't be mentioned in it.

Reply:
Yes, the docstring is intended for the public API. Maybe make the comment more explicit, e.g.

Note that in addition to the use case described in the docstring, `getfixturevalue()` is also called by pytest itself during item setup to evaluate the fixtures that are requested statically (using function parameters, autouse, etc).

Also, let's have an empty line between the comment and the code to make it clear it is not a comment for the next line but for the function.